ORE_EEG_2.py

```diff
@@ -5,24 +5,180 @@
 from PIL import Image
 from tempfile import NamedTemporaryFile
 import random
+import asyncio
+import pathlib
+import websockets # pip install websocket-client
+import json
+import ssl
+import os
+from dotenv import load_dotenv # pip install python-dotenv
 
 # Placeholder function for EEG setup and trigger recording
+load_dotenv()
+headset_info = {} # update this with the headset info
+
+ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+localhost_pem = pathlib.Path(__file__).with_name("cert.pem")
+ssl_context.load_verify_locations(localhost_pem)
```

Comment:
can you also add where in the documentation if mentions to include cert.pem?

Reply:
> Please note that Cortex only supports the WebSocket Secure (wss) protocol. It doesn't support the plain WebSocket (ws) protocol without encryption.

> The Cortex web socket server uses a self-signed certificate. When you install Emotiv softwares, the installer has already installed the Emotiv Root CA file and ask the system to trust it, so most of the time, your application don't need to configure anything else.

[docs](https://emotiv.gitbook.io/cortex-api/connecting-to-the-cortex-api)

Reply:
why are we passing in a `cert.pem` file if it's saying that it doesn't need it? Do you have reference to where you got this code from? Have you tried running this code without the `cert.pem` file and without the SSLContext?

Reply:
i think that means the launcher itself because the section after that is talking about using a raspberry pi as the wss server. it wasn't working without the `cert.pem` file